src/plugins/data/public/ui/filter_bar/index.tsx

```diff
@@ -26,3 +26,12 @@ export const FilterItem = (props: FilterItemProps) => (
     <LazyFilterItem {...props} />
   </React.Suspense>
 );
+
+import type { FilterBarProps } from './filter_bar';
+
+const LazyFilterBar = React.lazy(() => import('./filter_bar'));
+export const FilterBar = (props: FilterBarProps) => (
```

Comment:
The modifications of the filter_bar are just experimental. Changes here are needed if we want to allow editing of filters in the alert rule